config/quickwit.yaml

```diff
@@ -20,20 +20,23 @@ version: 0
 # - for its gRPC service (TCP)
 # - for its Gossip cluster membership service (UDP)
 #
-# All three services are bound to the same host and different port.
+# All three services are bound to the same host and a different port. The host can be an IP address or a hostname.
 #
 # Default HTTP server host is `127.0.0.1` and default HTTP port is 7280.
 #
 # listen_address: 127.0.0.1
 # rest_listen_port: 7280
 #
+# IP address advertised by the node, i.e. the IP address that peer nodes should use to connect to the node for gossip and gRPC.
+# advertise_address: 192.168.0.42
```

Comment:
Can you describe what the default value is here?

Reply:
Yes, I can elaborate.

Reply:
and that it can be overriden by `QW_ADVERTISE_ADDRESS`